docs/podman-create.1.md

```diff
@@ -23,6 +23,10 @@ The initial status of the container created with **podman create** is 'created'.
    Add a line to /etc/hosts. The format is hostname:ip.  The **--add-host**
 option can be set multiple times.
 
+**--annotation**=[]
```

Comment:
An example would be goo.

Reply:
Do we need to (or do we already) note the default ContainerType annotation being set to sandbox and what/how that's used?

Reply:
I think it's mostly used for kata containers... it might also have relevance to Kube under CRI-O, but here it's purely to tell Kata to make a new VM for the container.

Reply:
Well in this case the Annotations they are after are for running the nvidia hook.